internal/node/npm_package_bin.bzl

```diff
@@ -38,12 +39,14 @@ def _inputs(ctx):
 def _impl(ctx):
     if ctx.attr.output_dir and ctx.outputs.outs:
         fail("Only one of output_dir and outs may be specified")
-    if not ctx.attr.output_dir and not ctx.outputs.outs:
-        fail("One of output_dir and outs must be specified")
+    if not ctx.attr.output_dir and not ctx.outputs.outs and not ctx.attr.stdout:
```

Comment:
we don't have to restrict it in this way, you could get outputs or output_dir in addition to stdio outputs

Reply:
But wouldn't you want at least one of them? The extra condition is on the `and not ctx.attr.stdout`